tests/e2e.test.ts

```diff
@@ -0,0 +1,12 @@
+import { setupNearEthAdapter } from "../examples/setup";
+
+describe("End To End", () => {
+  it("Runs the Send ETH Tx", async () => {
+    const evm = await setupNearEthAdapter();
+    await evm.signAndSendTransaction({
+      to: "0xdeADBeeF0000000000000000000000000b00B1e5",
+      // THIS IS ONE WEI!
+      value: 1n,
+    });
+  });
```

Comment:
No `expect`, so this just assumes that no panic occurs?

Reply:
I guess we could change the method to return the tx hash and expect that it's not undefined.

Reply:
Actually. It might be even better to check the recipient balance before and after. Assert that the balance was increased by 1 wei.